mozetl/shield/utils.py

```diff
@@ -17,7 +17,7 @@ def etl_job(sc, sqlContext, submission_date=None, save=True):
 
         transformed_pings = transform_func(sqlContext, pings)
 
-        if save:
+        if type(save) == bool and save is True:
```

Comment:
I had started to expand this functionality to accept save value of `True`, `overwrite`, or `____`? - i.e., whatever `write.mode()` does **NOT** overwrite existing records.

I'm worried that calling an `etl_job` twice from `privacy_prefs.py` with the same `s3_path` means the 2nd job will overwrite the first job's records?